app/ts/utils/augurUtils.ts

```diff
@@ -33,3 +35,19 @@ export const getOutcomeNamesAndNumeratorCombinationsForMarket = (marketType: Mar
 
 // todo, make path typesafe
 export const getUniverseUrl = (universe: AccountAddress, path: string) => `/#/${ path }?universe=${ universe }`
+
+// https://github.com/AugurProject/augur/blob/bd13a797016b373834e9414096c6086f35aa628f/packages/augur-core/src/contracts/reporting/Market.sol#L384C51-L384C91
+export const requiredState = (allStake: bigint, stakeInOutcome: bigint) => (2n * allStake) - (3n * stakeInOutcome)
```

Comment:
Not sure if you meant `state` here or `stake`, but based on the parameters I am guessing `stake`.
```suggestion
export const requiredStake = (allStake: bigint, stakeInOutcome: bigint) => (2n * allStake) - (3n * stakeInOutcome)
```